test/ggsn_SUITE.erl

```diff
@@ -468,17 +468,27 @@ create_pdp_context_request_resend() ->
 create_pdp_context_request_resend(Config) ->
     S = make_gtp_socket(Config),
 
+    CntId = [socket,'gtp-c',irx,rx,v1,create_pdp_context_request,count],
+    DupId = [socket,'gtp-c',irx,rx,v1,create_pdp_context_request,duplicate],
+    Cnt0 = get_value(exometer:get_value(CntId)),
+    Dup0 = get_value(exometer:get_value(DupId)),
+
     {GtpC, Msg, Response} = create_pdp_context(S),
+    ?equal(1, get_value(exometer:get_value(CntId)) - Cnt0),
     ?equal(Response, send_recv_pdu(S, Msg)),
     ?equal([], outstanding_requests()),
 
     delete_pdp_context(S, GtpC),
+    ?equal(1, get_value(exometer:get_value(DupId)) - Dup0),
 
     ok = meck:wait(?HUT, terminate, '_', ?TIMEOUT),
     ?match(0, meck:num_calls(?HUT, handle_request, ['_', '_', true, '_'])),
     meck_validate(Config),
     ok.
 
+get_value({ok, DPs}) -> proplists:get_value(value, DPs, -1);
```

Comment:
pgw_SUITE already has a limited exometer check, please use the mechanism for this check and sync exmeter checks in pgw and ggsn SUITE

Reply:
https://github.com/travelping/ergw/blob/master/test/pgw_SUITE.erl#L858